src/components/text/text.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import './text.css';
+
+import data from '../../data';
+
+const Text = ({ pageName, controlHeader }) => {
+  const pageData = data[pageName];
+  let dataKey = '';
+
+  switch (controlHeader) {
+    case 'what':
+      dataKey = 'What?';
+      break;
+    case 'how':
+      dataKey = 'How?';
+      break;
+    case 'more-information':
+      dataKey = 'More Information';
+      break;
+    default:
+      break;
+  }
+
+  const textData = pageData[dataKey];
+  let text;
```

Comment:
I would use list name or something like that, text is not semantic

Reply:
It's not about the list name, but about the text that is shown below the header. Maybe 'PageContent' would be a better choice?

Reply:
whatever you like, but text is too common

Reply:
Fixed